### PR TITLE
 fix(claiming): use getUTCSeconds instead of getSeconds

### PR DESCRIPTION
**Pull request by Tommytrg**

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -551,7 +551,7 @@ export function calculateVesting(vestingInfo, amount, genesisDate) {
     .map((_, index) => {
       const date = new Date(genesisDate)
       date.setSeconds(
```
Comment (tmpolaczyk) [inline src/utils.js:553]:
```suggestion
      date.setUTCSeconds(
```

File: tests/unit/src/utils.spec.js [hunk 2]
```diff
@@ -48,7 +48,7 @@ describe('calculateTimeAgo', () => {
   it('should end with "ago" copy', () => {
     const date = new Date()
 
-    date.setSeconds(date.getSeconds() - 4)
+    date.setSeconds(date.getUTCSeconds() - 4)
```
Comment (tmpolaczyk) [inline tests/unit/src/utils.spec.js:51]:
```suggestion
    date.setUTCSeconds(date.getUTCSeconds() - 4)
```

File: src/utils.js [hunk 3]
```diff
@@ -550,8 +550,8 @@ export function calculateVesting(vestingInfo, amount, genesisDate) {
     .fill(0)
     .map((_, index) => {
       const date = new Date(genesisDate)
```
Comment (tmpolaczyk) [inline src/utils.js:552]: `new Date` also uses local time, not UTC

File: src/utils.js [hunk 4]
```diff
@@ -595,3 +596,14 @@ function factor(amount, base = 10, exp = 100) {
...
+export function convertDateToUtc (date = new Date()) {
+  return Date.UTC(
+    date.getUTCFullYear(),
+    date.getUTCMonth(),
+    date.getUTCDate(),
+    date.getUTCHours(),
+    date.getUTCMinutes(),
+    date.getUTCSeconds()
+  )
+}
```
Comment (Tommytrg) [inline src/utils.js:609]:
```suggestion
```

File: src/utils.js [hunk 5]
```diff
@@ -549,9 +549,10 @@ export function calculateVesting(vestingInfo, amount, genesisDate) {
   return Array(numberOfSteps)
     .fill(0)
     .map((_, index) => {
-      const date = new Date(genesisDate)
-      date.setSeconds(
-        date.getSeconds() + delay + cliff + installmentLength * index,
+      const date = convertDateToUtc(new Date(genesisDate))
```
Comment (Tommytrg) [inline src/utils.js:552]:
```suggestion
      const date = new Date(genesisDate)
```

File: src/utils.js (hunk 3 above)
Comment (aesedepece) [reply src/utils.js:553]: I agree that using `Date` here offers no advantage. We can simply go with:
```js
const date = genesisDate + delay + cliff + installmentLength * index
```

**Review by aesedepece — CHANGES_REQUESTED** [review]

File: tests/unit/src/utils.spec.js [hunk 6]
```diff
@@ -49,7 +49,7 @@ describe('calculateTimeAgo', () => {
   it('should end with "ago" copy', () => {
     const date = new Date()
 
-    date.setSeconds(date.getSeconds() - 4)
+    date.setUTCSeconds(date.getUTCSeconds() - 4)
```
Comment (tmpolaczyk) [inline tests/unit/src/utils.spec.js:52]: This doesn't need to be changed, it's just a test that checks if dates from the past are displayed as "ago"

**Review by gabaldon — APPROVED** [review]

**Review by aesedepece — APPROVED** [review]

**Pull request merged**